pyupgrade.py

```diff
@@ -2480,6 +2543,8 @@ def _fix_file(filename: str, args: argparse.Namespace) -> int:
         contents_text = _fix_py3_plus(contents_text)
     if args.min_version >= (3, 6):
         contents_text = _fix_py36_plus(contents_text)
+    if args.min_version >= (3, 8):
+        contents_text = _fix_py36_plus(contents_text)
```

Comment:
each parse / unparse adds a few 100ms to rewriting -- ideally we'd combine these in existing classes

Reply:
Sure